README.md

```diff
@@ -1,72 +1,95 @@
 [![CI](https://travis-ci.org/darwinia-network/darwinia.svg)](https://travis-ci.org/darwinia-network/darwinia])
 [![License](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 
-
-# ![Logo](https://github.com/darwinia-network/rfcs/raw/master/logo/darwinia.png)
-
-**As an open cross-chain bridge protocol based on Substrate, Darwinia aims to build the Internet of Tokens, including decentralized tokens swap, exchange, and market.**
-
-👉 *The official Documents for [The Darwinia Network](https://darwinia.network/)*<br>
-👉 *The official Documents for [The Crab Network](https://docs.crab.network/)*<br>
-👉 *The official communication zoom, if you have any question? [Darwinia Room](https://matrix.to/#/#darwinia:matrix.org)*<br>
+![Logo](https://github.com/darwinia-network/rfcs/raw/master/logo/darwinia.png)
+
+# Darwinia
+Implementation of a https://darwinia.network node in Rust based on the **Substrate** framework.
+
+This repository contains runtimes for the **[Darwinia](https://darwinia.network)** and **[Crab](https://crab.network)** networks.
+
+## Resources
+
+### Documents
+- [Darwinia Network Docs](https://docs.darwinia.network)
+- [Crab Network Docs](https://docs.crab.network)
+
+### Social Media
```

Comment:
We can get these Social Media from websites, only the channel for developers is needed here.